README.md

```diff
@@ -20,12 +19,14 @@ locally, with preconfigured connections.
 
 At present, these services are: postgres, redis, and the rails application.
 
-To start the application, run `docker compose up`.  Adding `-d` will free your
-terminal after the service boots.  After the application starts, the site can
+To start the application, run `docker compose up`. Adding `-d` will free your
+terminal after the service boots. After the application starts, the site can
 be accessed at `http://localhost:3000`.
 
+To set up the database the first time run, `docker compose run --rm stocks bin/rails db:setup`
```

Comment:
Within the container, this shouldn't be necessary -- the entrypoint script handles making sure the database is up to date and migrated.

it's necessary when running _outside_ docker, but we don't have an instruction block for that.